src/Core/Localization/CLDR/Locale.php

```diff
@@ -310,11 +318,21 @@ public function getCurrencyData($currencyCode)
     /**
      * Get CLDR data of all currencies.
      *
-     * @return currencyData[]
+     * @return CurrencyData[]
      *                        Data of all currencies
      */
     public function getAllCurrencies()
     {
         return $this->currencies;
     }
+
+    /**
+     * Get CLDR data of all territories.
+     *
+     * @return TerritoryData[] Data of all territories
+     */
+    public function getAllTerritories(): array
```

Comment:
Shouldn't the interface be updated as well Also I think this can be considered a BC break since the core now needs CLDR Locale to expose `getAllTerritories`

By the way why `getAllTerritories` and not simply `getTerritories` ?

Reply:
You're right for adding the method in interface. :heavy_check_mark: But I'm not agree with changing the name method, it's more coherent relative to `getAllNumberSymbols` or `getAllCurrencies`.

Reply:
@jolelievre I added the label "BC Break" and a note with the BC Break explained.